src/shared/models/screen_prop.ts

```diff
@@ -2,7 +2,7 @@ import { CompositeNavigationProp, RouteProp } from '@react-navigation/native';
 import { StackNavigationProp } from '@react-navigation/stack';
```

Comment:
i don't use this file much i just commented out some stuff in case it made vscode angry